src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputWhitespace.java

```diff
@@ -85,6 +85,10 @@ private int nonVoid()
     private void testCasts()
     {
         Object o = (Object) new Object(); // ok
+        o = (java.lang.
+                Long)o; // error
+        o = (java.lang.
+                Long) o; // ok
```

Comment:
I do not like amount of tests are updated , so you force me to verify all changes, too much.
Please create separate input file with with cases, or move them to the end of this file to have changes only in test that are affected in functional way.